examples/simple-game-server/main.go

```diff
@@ -787,8 +795,12 @@ func listContains(s *sdk.SDK, listName string, value string) (string, error) {
 	ok, err := s.Alpha().ListContains(listName, value)
 	if err != nil {
 		log.Printf("Error getting List %s contains value %s: %s", listName, value, err)
+		return "ERROR: " + strconv.FormatBool(ok) + "\n", err
 	}
-	return strconv.FormatBool(ok), err
+	if ok {
+		return "SUCCESS: " + strconv.FormatBool(ok) + "\n", nil
+	}
+	return "ERROR: " + strconv.FormatBool(ok) + "\n", err
```

Comment:
```suggestion
		return "SUCCESS: " + strconv.FormatBool(ok) + "\n", nil
```

I think this would be better - since it's a true/false as to whether if it's found. It's not really an ERROR.

Reply:
Done!

Reply:
`ListContains false` test fails with this change.

Shall we modify it as follows?
```
	if ok {
		return "SUCCESS: " + strconv.FormatBool(ok) + "\n", nil
	} else {
		log.Printf("Error getting List %s contains value %s: %s", listName, value, err)
		return "ERROR: " + strconv.FormatBool(ok) + "\n", err
	}
```

Reply:
Latest commit satisfies the edge cases. Please review it and share your thoughts.